element_calcium_imaging/imaging.py

```diff
@@ -1544,6 +1544,120 @@ def make(self, key):
             raise NotImplementedError("Unknown/unimplemented method: {}".format(method))
 
 
+@schema
+class ProcessingQualityMetrics(dj.Computed):
+    """Quality metrics used to evaluate the results of a calcium imaging analysis pipeline.
+
+    Attributes:
+        Segmentation (foreign key): Primary key from Segmentation.
+    """
+
+    definition = """
+    -> Segmentation
+    """
```

Comment:
How about we break this into two `Computed` tables without any `Part` tables?  Having the foreign key reference to only `Segmentation` would mean that `ProcessingQualityMetrics` could run after an entry is in `Segmentation` but before it is in `Fluorescence` and then the `make` method would throw an error.  Or we could update the `key_source`.

Reply:
I like the idea of separating them, more flexible.

Reply:
Perfect, how about the following names: `MaskQualityMetrics` and `FluorescenceQualityMetrics`?

Reply:
Another option is to make the attributes of FluorescenceTraceMetrics nullable.

Reply:
> Perfect, how about the following names: `MaskQualityMetrics` and `FluorescenceQualityMetrics`?

I remove `Quality` from the part table names on purpose because in the backed the master table name and the part table name are merged, resulting in mysql error.